### ISSUE TITLE
Add possibility to change overlay window gravity 

### ISSUE THREAD

**Issue by gercee** [skydoves/Balloon#347]
**Is your feature request related to a problem?**
Yes

The overlay window gravity is always centered, and we do have a need to have it positioned at the top.

**Describe the solution you'd like:**

We did local testing by adding the parameter into the builder and using the current gravity `Gravity.CENTER` as default and introduced another setter to be able to change it. And this doesn't break the current functionality and add extra possibility to configure the overlay needed for other use-cases.

```
@set:JvmSynthetic
public var overlayGravity: Int = Gravity.CENTER
```

```
/** sets the balloon overlay gravity. */
public fun setOverlayGravity(gravity: Int): Builder = apply {
  this.overlayGravity = gravity
}
```

```
private fun showOverlayWindow(vararg anchors: View) {
  if (builder.isVisibleOverlay) {
    val mainAnchor = anchors[0]
    if (anchors.size == 1) {
      overlayBinding.balloonOverlayView.anchorView = mainAnchor
    } else {
      overlayBinding.balloonOverlayView.anchorViewList = anchors.toList()
    }
    overlayWindow.showAtLocation(mainAnchor, builder.overlayGravity, 0, 0)
  }
}
```

The requirement is to use the keyboard while the tooltip is open. We are resizing the overlay window and we want the overlay gravity to stick to the top and not to the center.

**Comment by skydoves** [comment]
Hey @gercee, thank you for reporting and suggesting this issue.
It's available on version `1.4.7-SNAPSHOT`, and this will be shipped in the next release. Thanks!

**Comment by skydoves** [comment]
This has been released in version `1.4.7`.
